rplugin/python3/defx/view.py

```diff
@@ -154,10 +149,12 @@ def redraw(self, is_force: bool = False) -> None:
         self._buffer.options['modifiable'] = False
         self._buffer.options['modified'] = False
 
-        self._vim.call('cursor', [prev_linenr, 0])
-
-        if prev:
-            self.search_file(prev['action__path'], prev['_defx_index'])
+        # TODO: How to set cursor position for other buffer when
```

Comment:
Unfortunately, nothing.
You must jump to buffer to change the position.

Reply:
I made a feature request, https://github.com/neovim/neovim/issues/10822#issue-483150362